.github/workflows/build.yml

```
@@ -451,6 +450,125 @@ jobs:
+    steps:
+      - name: Download binary
+        uses: actions/download-artifact@v4
+        with:
+          name: binary-x86_64-unknown-linux-musl
+          path: dist/
+
+      - name: Make executable
+        run: chmod +x dist/unified-hifi-linux-x64
+
+      - name: Start server
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2046:warning:11:16: Quote this to prevent word splitting

(shellcheck)